sysvinit/freebsd/cloudconfig

```diff
@@ -22,4 +22,7 @@ cloudconfig_start()
 }
 
 load_rc_config $name
+
+: ${cloudconfig_enable="NO"}
```

Comment:
I'm under the impression that these lines are not necessary. The default if underfine is already NO.

Reply:
in that case, we should take that up with the freebsd maintainer.

Reply:
> I'm under the impression that these lines are not necessary. The default if underfine is already NO.

Hi,

The default is actually unset, and you get a nice warning:

```
/usr/local/etc/rc.d/cloudconfig: WARNING: $cloudconfig_enable is not set properly - see rc.conf(5).
``` 

Because checkyesno wants an explicit yes/no, and we don't assume either way in the framework.

Reply:
Thanks @keyz182 for the information :-)

Reply:
I think you mean @kevans91, it's been nearly 9 years since I commited anything here :p